wikidata_game/stage-game.php

```diff
@@ -0,0 +1,47 @@
+<?php
+    if(!isset($_SERVER['HTTP_X_GITHUB_EVENT'])){
+        exit;
+    }
+
+    if($_SERVER['HTTP_X_GITHUB_EVENT'] != 'pull_request'){
+        exit;
+    }
+
+    $logfile_path = '../logs/staging.log';
+    $datetime_string = date('c');
+
+    $payload = json_decode($_POST["payload"]);
+    $actions = ['synchronize', 'opened', 'reopened'];
+
+    $branch = $payload->pull_request->head->ref;
+    // Check if current branch is a game branch
+    $is_game_branch = substr($branch, 0, 5) === 'game-';
+
+    if($payload->action === 'closed'){
+        $root = $_SERVER['DOCUMENT_ROOT'];
+        $status = NULL;
+        $output = [];
+        exec('rm ' . $root . 'stage/' . $branch . '-api.php', $output, $status);
```

Comment:
I think this needs to be escaped to help prevent it being use to execute arbitrary commands